### DISCUSSION TITLE
[功能建议]动态地址参数清理

### DISCUSSION THREAD

**Discussion by NightSharp** [the1812/Bilibili-Evolved#2350] [功能建议 / Ideas]
我的单个动态的地址后总会显示一个"?tab=2"，有办法去掉吗

**Comment by the1812** [comment]
tab=2 是切换到评论, 不算是无意义参数

**Reply by NightSharp** [reply]
不带上参数的话也能正常打开页面
能不能像视频那样默认不带?p=啥的
可能在动态首页需要这个参数，但我只想点击打开单个动态时的地址不带这个参数
比如从相簿点开动态时就没这个参数

**Reply by the1812** [reply]
> 能不能像视频那样默认不带?p=啥的

视频如果带了 `?p=` 也不会清理

> 可能在动态首页需要这个参数，但我只想点击打开单个动态时的地址不带这个参数

这个参数就是单个动态里用的

你非要去掉可以等 2.0 发布

**Reply by NightSharp** [reply]
> 
> 
> > 能不能像视频那样默认不带?p=啥的
> 
> 视频如果带了 `?p=` 也不会清理
> 
> > 可能在动态首页需要这个参数，但我只想点击打开单个动态时的地址不带这个参数
> 
> 这个参数就是单个动态里用的
> 
> 你非要去掉可以等 2.0 发布

谢谢解惑

**Comment by Geeyun-JY3** [comment]
这是个有用参数，定位打开动态后下方打开评论 tab 的，所以没清。如果参数是 `?tab=1`，打开动态后的下方打开的就是转发 tab 了。

想获得无这参数的干净链接可以开启 `动态链接复制` 功能，更多菜单里点复制链接拿到的就是干净链接了。

**Reply by NightSharp** [reply]
不带上参数的话也能正常打开页面
能不能像视频那样默认不带?p=啥的
可能在动态首页需要这个参数，但我只想点击打开单个动态时的地址不带这个参数
比如从相簿点开动态时就没这个参数

**Comment by the1812** [comment]
现在好像没这参数了
~~又是鸽到官方实现~~

反正插件我先放这了
[urlParamsClean.feedsTypeClean.zip](https://github.com/the1812/Bilibili-Evolved/files/9383023/urlParamsClean.feedsTypeClean.zip)

**Reply by Geeyun-JY3** [reply]
`?tab=1` 这参数倒是还能用（以前切换 tab 这个参数是不是会跟着变来着？）
